PowerUp/app/src/main/java/powerup/systers/com/SplashActivity.java

```diff
@@ -7,28 +7,17 @@
 
 public class SplashActivity extends AppCompatActivity {
 
-    private Handler handler;
-    private Runnable runnable;
-
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        handler = new Handler();
-        runnable = new Runnable() {
+        new Handler().postDelayed(new Runnable() {
             @Override
             public void run() {
                 Intent intent = new Intent(SplashActivity.this, StartActivity.class);
                 startActivity(intent);
                 overridePendingTransition(R.animator.fade_in, R.animator.fade_out);
                 finish();
             }
-        };
-        handler.postDelayed(runnable,1000);
-    }
-
-    @Override
-    protected void onDestroy() {
-        super.onDestroy();
-        handler.removeCallbacks(runnable);
```

Comment:
Since the PR just concerns with changing documentation, affecting parts of code is not desired. Revert these changes.

Reply:
@Rimjhim28  Yes I noticed this. Initially It did not have any of those changes, but when I was trying to squash the commits, I think I did something wrong because it added those changes too.